doc/widgets/networktwitter.rst

```diff
@@ -0,0 +1,30 @@
+================
+Twitter User Graph
+================
+
+.. figure:: icons/network-twitter.png
```

Comment:
This is not a proper icon. A proper doc icon includes the surrounding colored bubble.